src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -231,7 +233,11 @@ public Config validate(Map<String, String> connectorConfigs) {
     SnowflakeConnectionService testConnection;
     try {
       testConnection =
-          SnowflakeConnectionServiceFactory.builder().setProperties(connectorConfigs).build();
+          SnowflakeConnectionServiceFactory.builder()
+                  .setProperties(connectorConfigs)
+                  .setNetworkTimeout(VALIDATION_NETWORK_TIMEOUT)
```

Comment:
Not required for this PR, but would be nice to have generic JDBC parameter passthrough akin to what we get in JDBC and Debezium connectors with the `connection.` and `database.` prefixes. Maybe we can file a feature request for this upstream.